test/e2e/serial/tests/operator_profile_update.go

```diff
@@ -0,0 +1,282 @@
+			testPod.Spec.Containers[0].Resources.Requests = corev1.ResourceList{
+				corev1.ResourceMemory: resource.MustParse("256Mi"),
+			}
+			err := fxt.Client.Create(context.TODO(), testPod)
+			Expect(err).ToNot(HaveOccurred())
+
+			err = wait.With(fxt.Client).Interval(10*time.Second).Steps(3).WhileInPodPhase(context.TODO(), testPod.Namespace, testPod.Name, corev1.PodPending)
+			if err != nil {
+				_ = objects.LogEventsForPod(fxt.K8sClient, testPod.Namespace, testPod.Name)
+			}
+			Expect(err).ToNot(HaveOccurred())
+
+			By(fmt.Sprintf("verify pod events contain the correct message %q", eventMsgInvalidTopology))
+			ok, err := verifyPodEvents(fxt, testPod, eventReasonFailedScheduling, eventMsgInvalidTopology)
+			Expect(err).ToNot(HaveOccurred())
+			if !ok {
+				_ = objects.LogEventsForPod(fxt.K8sClient, testPod.Namespace, testPod.Name)
+			}
+			Expect(ok).To(BeTrue(), "failed to find the expected event with Reason=%q and Message contains %q", eventReasonFailedScheduling, eventMsgInvalidTopology)
+
+			By("restore NROP configuration and verify the pod goes running")
+			restoreNROP(fxt, *nropObjInitial)
+			updatedPod, err := wait.With(fxt.Client).Timeout(5*time.Minute).ForPodPhase(context.TODO(), testPod.Namespace, testPod.Name, corev1.PodRunning)
+			if err != nil {
+				_ = objects.LogEventsForPod(fxt.K8sClient, updatedPod.Namespace, updatedPod.Name)
+			}
+			Expect(err).ToNot(HaveOccurred())
+		})
+
+		FIt("[tier1][test_id:47611] should make a guaranteed pod pending", func() {
+			By("create guaranteed pod and expect it to keep pending")
+			testPod = objects.NewTestPodPause(fxt.Namespace.Name, "testpod")
+			testPod.Spec.SchedulerName = serialconfig.Config.SchedulerName
+			testPod.Spec.Containers[0].Resources.Limits = corev1.ResourceList{
+				corev1.ResourceCPU:    resource.MustParse("8"),
+				corev1.ResourceMemory: resource.MustParse("256Mi"),
+			}
+
+			err := fxt.Client.Create(context.TODO(), testPod)
+			Expect(err).ToNot(HaveOccurred())
+
+			err = wait.With(fxt.Client).Interval(10*time.Second).Steps(3).WhileInPodPhase(context.TODO(), testPod.Namespace, testPod.Name, corev1.PodPending)
+			if err != nil {
+				_ = objects.LogEventsForPod(fxt.K8sClient, testPod.Namespace, testPod.Name)
+			}
+			Expect(err).ToNot(HaveOccurred())
+
+			By(fmt.Sprintf("verify pod events contain the correct message %q", eventMsgInvalidTopology))
+			ok, err := verifyPodEvents(fxt, testPod, eventReasonFailedScheduling, eventMsgInvalidTopology)
+			Expect(err).ToNot(HaveOccurred())
+			if !ok {
+				_ = objects.LogEventsForPod(fxt.K8sClient, testPod.Namespace, testPod.Name)
+			}
+			Expect(ok).To(BeTrue(), "failed to find the expected event with Reason=%q and Message contains %q", eventReasonFailedScheduling, eventMsgInvalidTopology)
+
+			By("verify NRTs has no changes")
+			//
+
+			By("restore NROP configuration and verify the pod goes running")
+			//restoreNROP(fxt, *nropObjInitial)
```

Comment:
why commented out?

Reply:
for testing and to make it faster to debug :) 
will fix it eventually.